wandb/sdk/internal/internal_api.py

```diff
@@ -1218,6 +1231,85 @@ def ack_run_queue_item(self, item_id: str, run_id: Optional[str] = None) -> bool
         result: bool = response["ackRunQueueItem"]["success"]
         return result
 
+    @normalize_exceptions
+    def create_default_resource_config(
+        self, entity_name: str, resource: str, config: str
+    ) -> Optional[Dict[str, Any]]:
+        mutation = gql(
+            """
+        mutation createDefaultResourceConfig(
+            $entityName: String!,
+            $resource: String!,
+            $config: JSONString!
+        ) {
+            createDefaultResourceConfig(input: {
+                entityName: $entityName,
+                resource: $resource,
+                config: $config
+            }) {
+                defaultResourceConfigID
+                success
+            }
+        }
+        """
+        )
+
+        response = self.gql(
+            mutation,
+            variable_values={
+                "entityName": entity_name,
+                "resource": resource,
+                "config": config,
+            },
+        )
+
+        if not response or not response.get("createDefaultResourceConfig", {}).get(
```

Comment:
Im unclear about whether this is the preferred pattern or introspection is. Since this only being used for tests I'm fine with this